tests/e2e/test_spyre_pc_scheduler_steps.py

```diff
@@ -1161,14 +1229,27 @@ def test_multi_chunk_partial_match(model: ModelInfo, backend: str,
     """
     monkeypatch.setenv("VLLM_SPYRE_CP_INTERLEAVE_STEPS", "0")
 
-    seqs_max_tokens = [2, 2]
-    prompts_lengths = [384, 384]
-    steps_add_reqs = [0, 0]
-    seeds = [0, 0]  # twice the same seed for a sequence of length 384
+    # twice the same seed for a sequence of length 384
     # the first sequence shares the same prefix of length 384 tokens
     # the second sequence shares the same prefix of length 254 tokens
     # hence sequence 1 shares the first 254 tokens with sequence 0
-    prefix_lens = [384, 254]
+
+    request1 = create_request_for_scheduler_test(
+        model=model,
+        request_id=0,
+        add_step=0,
+        max_tokens=2,
+        prompt=random_prompt(model=model, seed=0, length=384),
+        use_golden_token_injection=True)
+
+    request2 = create_request_for_scheduler_test(
+        model=model,
+        request_id=1,
+        add_step=0,
+        max_tokens=2,
+        prompt=request1.request.prompt_token_ids[0:254] +
+        random_prompt(model=model, seed=0, length=384 - 254),
+        use_golden_token_injection=True)
```

Comment:
oh this is makes things actually so much simpler, no need to have the dedicated logic to create partial prefix prompt in the functions, just for a simple test. 

But I would put prompt 2 in a dedicated variable, for readability and highlighting that it is partial prefix prompt

```suggestion
    prompt2= request1.request.prompt_token_ids[0:254] +
        random_prompt(model=model, seed=0, length=384 - 254)
    request2 = create_request_for_scheduler_test(
        model=model,
        request_id=1,
        add_step=0,
        max_tokens=2,
        prompt=prompt2,
        use_golden_token_injection=True)
```

Reply:
Thanks @sducouedic! 

I went ahead and did this for all the test cases, they're all more readable now 🚀🚀🚀